timescaledb/how-to-guides/install-timescaledb/installation-macos.md

```diff
@@ -0,0 +1,136 @@
+# Install self-hosted TimescaleDB on macOS systems
+You can host TimescaleDB yourself on your Apple macOS system.
+These instructions use a Homebrew installer on these versions:
+*   Apple macOS 10.15 Catalina
+*   Apple macOS 11 Big Sur
+*   Apple macOS 12 Monterey
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than Homebrew, you
+could encounter errors following these instructions. It is safest to remove any
+existing PostgreSQL installations before you begin. If you want to keep your
+current PostgreSQL installation, do not install TimescaleDB using this method.
+using this method.
+TimescaleDB using this method.
+[Install from source](/how-to-guides/install-timescaledb/installation-source/)
```

Comment:
I wonder how easy it would be to get a fresh MacOS install? We seem to have pretty consistent reports of TimescaleDB having issues on MacOS and most of the time I think it's because of the preinstalled version that tends to come with it. (or gets installed when you install developer tools maybe). I remember having issue too (16 months ago now) and having to find lots of help on linking Homebrew to the correct binary, etc.

Reply:
We test-run the installation on MacOS nightly in CI here is the code run during that test: https://github.com/timescale/timescaledb/blob/master/.github/workflows/homebrew.yaml

Reply:
I think the homebrew packaging is currently flawed since we don't have the major postgres version as part of the package name. So when a new postgres version is released that we don't yet support you cannot install timescaledb out of the box and you have to take extra-steps to ensure the extension gets build agaisnt the correct version.

Reply:
I didn't have any issues on my local machine.

Reply:
it is working now but it was broken when pg14 released but we hadnt released a timescaledb version with pg14 yet.  so at certain times it can be a bit more challenging to get it working. I think having separate homebrew packages for different postgres major versions would fix this specific issue (e.g. have packages timescaledb-postgresql13, timescaledb-postgresql14, ...).